### PR TITLE
add issue templates

### LINKED ISSUE #732: Setup issue templates
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

### PR DESCRIPTION
**Pull request by Emilgardis**
resolves #732

### CODE REVIEW

**Comment by Alexhuszagh** [comment]
Any reason for not using Markdown for the issue templates? It can be a fair amount more readable. An example would be:

```markdown
---
name: Bug report
about: Create a report to help us improve.
title: "[BUG]"
labels: bug
---

## Description

Please include a clear and concise description of the bug.

## Prerequisites

Here are a few things you should provide to help me understand the issue:

- Rust version: `rustc -V`
- cross version:
- cross targets used:

## Test case

Please provide a short, complete example of the issue.

## Additional Context
Add any other context about the problem here.

```

**Comment by Emilgardis** [comment]
https://github.com/cross-rs/cross/blob/issue-templates/.github/ISSUE_TEMPLATE/b_issue_report.yml

**Comment by Emilgardis** [comment]
> Any reason for not using Markdown for the issue templates?

Wanted to try out the beta feature for forms :D I think I like it

**Review by Alexhuszagh — APPROVED** [review]

**Comment by Emilgardis** [comment]
bors cancel

File: .github/ISSUE_TEMPLATE/b_issue_report.yml [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        description: The version of cross given with `cross -V` or `cross --version`
+        placeholder: cross 0.4.2 (c8df353 2025-01-01)
+
+  - type: textarea
+    id: mcve
+    attributes:
+      label: Example
+      description: Please provide a short, complete example of the issue if possible.
+      placeholder: |
+          ````sh
```
Comment (Emilgardis) [inline .github/ISSUE_TEMPLATE/b_issue_report.yml:129]: ...

**Pull request merged**